src/main/java/org/zalando/nakadi/repository/kafka/KafkaLocationManager.java

```diff
@@ -99,9 +99,7 @@ private void updateBrokers() {
     }
 
     public Properties getKafkaConsumerProperties() {
-        final Properties consumerProps = (Properties) kafkaProperties.clone();
-        consumerProps.put("fetch.message.max.bytes", kafkaSettings.getFetchMessageMaxBytes());
```

Comment:
@adyach yes, but this is not working. I get this in the logs: `[org.apache.kafka.clients.consumer.ConsumerConfig] --- The configuration fetch.message.max.bytes = 2000000 was supplied but isn't a known config.`

I think we could do this as a separate ticket, where we increase the max event size from 999,000 bytes to 1,000,000 bytes